commons/src/main/java/io/ballerina/stdlib/graphql/commons/utils/SdlSchemaStringGenerator.java

```diff
@@ -189,7 +189,8 @@ private static String createFields(Type type) {
     private static String createEnumValues(Type type) {
         List<String> enumValues = new ArrayList<>();
         for (EnumValue enumValue : type.getEnumValues()) {
-            enumValues.add(getFormattedString(ENUM_VALUE_FORMAT, enumValue.getName(), createDeprecate(enumValue)));
+            enumValues.add(getFormattedString(ENUM_VALUE_FORMAT, createFieldDescription(enumValue.getDescription()),
```

Comment:
Shall we add a test for this?

Reply:
The test will add to the GraphQL tool. Currently, we can not add tests for this in the GraphQL module. Tracked in https://github.com/ballerina-platform/ballerina-standard-library/issues/3578